### PR TITLE
SciKit-Data recipe added.

### PR DESCRIPTION
**Pull request by xmnlab**

### CODE REVIEW

**Comment by xmnlab** [comment]
@ocefpaf Do you know why this problem happens?

```sh

The following specifications were found to be in conflict:
  - numpy 1.9* -> python 2.6*
  - python 3.6*

```

File: recipes/scikit-data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k and py<34]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/scikit-data/meta.yaml]: It looks like `python` and `setuptools` are the only `build` requirements.

File: recipes/scikit-data/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  summary: 'The propose of this library is to allow the data analysis process more easy and automatic.'
+  description: |
+    This library offers functions to manipulate, clean and visualize data in a easy way.
+  doc_url: http://skdata.readthedocs.io/
+  dev_url: https://github.com/OpenDataScienceLab/skdata
+
+extra:
+  recipe-maintainers:
+    - xmnlab
+    - ocefpaf
```
Comment (synapticarbors) [inline recipes/scikit-data/meta.yaml]: @ocefpaf - are you ok with being a co-maintainer on this project?

**Comment by ocefpaf** [comment]
> @ocefpaf Do you know why this problem happens?

Ivan I notice this bug yesterday. It should not happen, but specifying a higher numpy version solves it, like you already found out.

File: recipes/scikit-data/meta.yaml (hunk 2 above)
Comment (ocefpaf) [inline recipes/scikit-data/meta.yaml]: @xmnlab is my colleague, don't worry :smile:

File: recipes/scikit-data/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/scikit-data/meta.yaml]: @xmnlab what @synapticarbors is saying here is that, if you don't import the module in `setup.py`, like to get the version, you don't need the run-time dependencies at build-time. This is odd for `pip` users b/c this distinction in the `pip` realm is `install_requires` vs `build_requires`.

I can see in your [setup.py](https://github.com/OpenDataScienceLab/skdata/blob/master/setup.py) that you don't import `scikit-data` itself, so it is save to remove everything but `python` and `setuptools` from the build section.

**Comment by ocefpaf** [comment]
@xmnlab this looks good! Any remaining suggestion can be done in the `feedstock`.
Let us merge this so I can show you what happens next.

**Pull request merged**